modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/CTMessageBusInterceptorTest.java

```diff
@@ -68,7 +72,26 @@ public static void setUpClass() {
 		Bundle bundle = FrameworkUtil.getBundle(
 			CTMessageBusInterceptorTest.class);
 
-		_bundleContext = bundle.getBundleContext();
+		BundleContext bundleContext = bundle.getBundleContext();
+
+		_testMessageListener = new TestMessageListener();
+
+		Destination destination = _destinationFactory.createDestination(
```

Comment:
Hi @Preston-Crary ,
I can not understand why you need a serial type destination. Serial type is actually async, with an async destination, how can we assure messagelistener's behavior?

Tina.

Reply:
It's working because of the Sync test rule on the test methods.  I only picked serial because that's the type subscription sender already uses.

Reply:
Made it DESTINATION_TYPE_SYNCHRONOUS.